src/Tools/Source/RunTests/Options.cs

```diff
@@ -138,10 +138,24 @@ internal class Options
             Architecture = architecture;
         }
 
+        private static string GetArchitecture() => RuntimeInformation.ProcessArchitecture switch
+        {
+            System.Runtime.InteropServices.Architecture.Arm => "arm",
+            System.Runtime.InteropServices.Architecture.Arm64 => "arm64",
+            System.Runtime.InteropServices.Architecture.Armv6 => "armv6",
+            System.Runtime.InteropServices.Architecture.LoongArch64 => "loongarch64",
+            System.Runtime.InteropServices.Architecture.Ppc64le => "ppc64le",
+            System.Runtime.InteropServices.Architecture.S390x => "s390x",
+            System.Runtime.InteropServices.Architecture.Wasm => "wasm",
+            System.Runtime.InteropServices.Architecture.X86 => "x86",
+            System.Runtime.InteropServices.Architecture.X64 => "x64",
+            _ => throw new NotImplementedException()
```

Comment:
This rough code path being copied three times bothers me a bit. Trying to think of a way to reduce it. 

This copy should be able to reference the one in `ILAsmUtilities.cs`. Or at least we should be able to put it in one location in that project and reference it here.

Reply:
Yeah, it bothered me as well. My only thought had been putting it in a separate file and source-including it in each place, and I wasn't sure that was actually better.

Reply:
I think we could simplify this whole method to `RuntimeInformation.ProcessArchitecture.ToString().ToLowerInvariant()`.

Reply:
Done.